static/ankersrv.js

```diff
@@ -97,13 +114,13 @@ $(function () {
         constructor({
             name,
             url,
-            badge=null,
-            open=null,
-            close=null,
-            error=null,
-            message=null,
-            binary=false,
-            reconnect=1000,
+            badge = null,
```

Comment:
There are a lot of seemingly unrelated changes in this file - not all of them not for the better, I have to say. Is this automatic reformatting, or a deliberate change?

I'd really appreciate it if we can avoid mixing semantic and stylistic changes, as it makes the PRs much harder to review. Optimally, general cleanup/refactor/reformatting should go in a separate PR, but at least in independent commits. Please? :smile:

Reply:
Automatic formatting that was unintentional. We designated style guides for python but not for html and javascript files so Prettier stepped in. I'll see if I can revert some of this for the PR.

Reply:
✅